cwiczenie3/person.php

```diff
@@ -98,4 +114,25 @@ public function getName()
         {
             return "Samochod";
         }
+}
+
+interface Animals
```

Comment:
Czemu liczba mnoga?

Reply:
tak jakoś widocznie miałem z tyłu głowy, że będzie tego więcej niż jedno, ale spoko wiem czemu pytasz ;)

Reply:
To popraw :)